Assets/Scripts/LevelTransitions/Object.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+
+namespace OperationBlackwell.LevelTransitions {
+	[CreateAssetMenu(fileName = "New Object", menuName = "Levels/TransitionObject")]
+	public class Object : ScriptableObject {
```

Comment:
While I know namespaces are a thing, and we use them, C# has an `Object` class too, that might be confusing. Can't we pick a different name? Especially since C# `Object` is in the `System` namespace and we have a few `using System` declarations in the code.

Reply:
I totally get where you are coming from, but never is `Object` used in C# as they always refer to it as `object` and this is ment to never be used outside of the `LevelTransitionController`. I know it does not make use of it now but it is there if we want to add start and end objects. And as you say yourself it is in a different namespace, this is one of the main reasons to even use namespaces in general having colliding names, and if it gives errors in files we just have to specify the correct `Object` there.

Reply:
That is true, but it might become confusing for other people to look at. It's fine by me, but please keep that in mind in the future.